run/README.md

```diff
@@ -9,71 +9,73 @@ This directory contains samples for [Google Cloud Run](https://cloud.run). [Clou
+|[System Packages](system-package/)| Using system packages with containers |
+|[Image Magick](image-processing/) | Event-driven image analysis & transformation |
+|[Manual Logging](logging-manual/) | Structured logging for Stackdriver |
+|[Local Troubleshooting](hello-broken/) | Broken services for local troubleshooting tutorial |
+|[Cloud SQL (MySQL)][mysql]        | Use MySQL with Cloud Run |
 
 For more Cloud Run samples beyond Java, see the main list in the [Cloud Run Samples repository](https://github.com/GoogleCloudPlatform/cloud-run-samples).
 
 ## Setup
 
 1. [Set up for Cloud Run development](https://cloud.google.com/run/docs/setup)
 
-2. Clone this repository:
+1. Clone this repository:
 
     ```
     git clone https://github.com/GoogleCloudPlatform/java-docs-samples.git
     ```
 
-    Note: Some samples in the list above are hosted in other repositories. They are noted with the symbol "&#10149;".
-
+1. In the samples's `pom.xml`, update the image field for the `jib-maven-plugin`
+with your Google Cloud Project Id:
+
+    ```XML
+    <plugin>
+      <groupId>com.google.cloud.tools</groupId>
+      <artifactId>jib-maven-plugin</artifactId>
+      <version>1.8.0</version>
+      <configuration>
+        <to>
+          <image>gcr.io/PROJECT_ID/SAMPLE_NAME</image>
+        </to>
+      </configuration>
+    </plugin>
+    ```
 
 ## How to run a sample locally
 
-1. [Install docker locally](https://docs.docker.com/install/)
-
-2. [Build the sample container](https://cloud.google.com/run/docs/building/containers#building_locally_and_pushing_using_docker):
+1. [Build the sample container using Jib](https://github.com/GoogleContainerTools/jib):
 
-    ```
-    export SAMPLE=<SAMPLE_NAME>
-    cd $SAMPLE
-    docker build --tag $SAMPLE .
+    ```Bash
+    mvn jib:build
     ```
 
-3. [Run containers locally](https://cloud.google.com/run/docs/testing/local)
+1. [Run containers locally](https://cloud.google.com/run/docs/testing/local) by
+replacing `PROJECT_ID` and `SAMPLE_NAME` with your values.
 
     With the built container:
 
-    ```
-    PORT=8080 && docker run --rm -p 8080:${PORT} -e PORT=${PORT} $SAMPLE
-    ```
-
-    Overriding the built container with local code:
-
-    ```
-    PORT=8080 && docker run --rm \
-        -p 8080:${PORT} -e PORT=${PORT} \
-        -v $PWD:/app $SAMPLE
+    ```Bash
+    PORT=8080 && docker run --rm -p 8080:${PORT} -e PORT=${PORT} gcr.io/PROJECT_ID/SAMPLE_NAME
     ```
 
     Injecting your service account key for access to GCP services:
 
-    ```
-    # Set the name of the service account key within the container
-    export SA_KEY_NAME=my-key-name-123
-
-    PORT=8080 && docker run --rm \
-        -p 8080:${PORT} \
-        -e PORT=${PORT} \
-        -e GOOGLE_APPLICATION_CREDENTIALS=/tmp/keys/${SA_KEY_NAME}.json \
-        -v $GOOGLE_APPLICATION_CREDENTIALS:/tmp/keys/${SA_KEY_NAME}.json:ro \
-        -v $PWD:/app $SAMPLE
+    ```Bash
+    PORT=8080 && docker run \
+       -p 9090:${PORT} \
```

Comment:
nit: Is this supposed to be 9090?

Reply:
Yes, based on https://cloud.google.com/run/docs/testing/local. I've added a note to clarify